lib/Controller/DocumentController.php

```diff
@@ -623,7 +721,7 @@ private function createWopiSessionForAuthUser(array $docInfo) : array {
 		$wopiSrc = $this->discoveryService->getWopiSrc($mimetype);
 		if (($allowEdit === true) && isset($wopiSrc['action'])
 				&& ($wopiSrc['action'] === 'edit' || $wopiSrc['action'] === 'view_comment')
-				&& ($this->isAllowedEditor($currentUser) === true) && ($version === '0')) {
+				&& ($this->isAllowedEditor($editorUid) === true) && ($version === 0)) {
```

Comment:
I guess `$version === 0` is intentional. Double-checking because now it's an integer, but it was a string before.